notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,67 @@
+# Integration of payment into checkout process
```

Comment:
This file should consist out of two main sections: Deployment (Docker, how to run/setup, env), Configuration (Adyen subscription)

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/45/commits/caeb4d82241e0af74252ff9bd352a679801572b2